linux/step01_centos7_ice_deps.sh

```diff
@@ -15,4 +15,18 @@ elif [ "$ICEVER" = "ice35-devel" ]; then
 	http://download.zeroc.com/Ice/3.5/el7/zeroc-ice-el7.repo
 
 	yum -y install ice ice-python-devel ice-java-devel ice-servers
+elif [ "$ICEVER" = "ice36" ]; then
+	cd /etc/yum.repos.d
+	wget https://zeroc.com/download/rpm/zeroc-ice-el7.repo
+
+	#for omero, to remove when we do not install from git
+	yum -y install git
```

Comment:
is there any reason for installing get as a part of ice dependencies? why not do it in https://github.com/ome/omero-install/pull/102/files#diff-49a05e9c183b6b84aa966891a004eb37R30